examples/sdk-app-example/src/main/AndroidManifest.xml

```diff
@@ -34,14 +34,6 @@
       android:name="com.bugsnag.android.API_KEY"
       android:value="your-api-key" />
```

Comment:
It'd be good to update this to a valid dummy API key, as currently this crashes the example app.